utils/data-migration/migrate-acbs-utilisation-data-to-tfm/README.md

```diff
@@ -0,0 +1,65 @@
+# ACBS Utilisation Data Migration to TFM
+
+This script migrates ACBS facility utilisation data to TFM by reading from a 
+JSON file and inserting the data into the SQL database.
+
+## JSON Input File Structure
+
+The script expects a JSON file with the following structure:
+
+```json
+[
+  {
+    "facilityId": "string",
+    "utilisation": number,
+    "fixedFeePremium": number
```

Comment:
Nit but because this is `json`, `number` is not valid - could we replace with example number literals?
```suggestion
    "utilisation": 100000,
    "fixedFeePremium": 1000
```